Update connectivity-settings.md (#116378)
Azure/azure-content: articles/synapse-analytics/security/connectivity-settings.md

- When public network access is **enabled**, you can connect to your workspace also from public networks. You can manage this feature both during and after your workspace creation. 

> [!IMPORTANT]
> This feature is only available to Azure Synapse workspaces associated with [Azure Synapse Analytics Managed Virtual Network](synapse-workspace-managed-vnet.md). However, you can still open your Synapse workspaces to the public network regardless of its association with managed VNet.
> 

> > When the public network access is disabled, access to GIT mode in Synapse Studio will not be blocked as long as the user has enough permission to access the integrated Git repo or the corresponding Git branch, which is by design. But please be assured that the publish button will not work as the access to Live mode is blocked by the firewall settings. 

Reviewer: ```suggestion
> When the public network access is disabled, access to GIT mode in Synapse Studio and commit changes won't be blocked as long as the user has enough permission to access the integrated Git repo or the corresponding Git branch. However, the publish button won't work because the access to Live mode is blocked by the firewall settings.
```
